### DISCUSSION TITLE
自定义 homepage

### DISCUSSION THREAD

**Discussion by violenttamper** [violentmonkey/violentmonkey#1270] [Q&A] [answered]
我希望在脚本管理界面点击脚本的主页时能跳转到安装脚本的网站，虽然大部分脚本实现了，不过还有部分脚本还是会跳转到奇怪的页面，比如“Picviewer CE+”这个脚本，在脚本管理界面点击“主页”，会跳转到 http://hoothin.com  这个网页，而非  https://greasyfork.org/zh-CN/scripts/24204-picviewer-ce  这个安装界面。还有 “网页限制解除(改)” 这个脚本点击“主页”会跳转到旧脚本的网页 https://cat7373.github.io/remove-web-limits/  而非 https://greasyfork.org/zh-CN/scripts/28497-remove-web-limits-modified  这个安装网页。希望能解决这个问题。

**Comment by tophf** [comment, marked as answer]
This is working as intended: Violentmonkey displays `@homepage` if it's present in the script.

**Reply by violenttamper** [reply]
我认为这里面是有问题，因为大多数脚本的主页都是安装脚本的网站，而那两个却是到了其他的页面，希望能重视一下，这个问题

**Reply by tophf** [reply]
There's no problem in Violentmonkey. It works correctly as intended. These two scripts define their own `@homepage`

![image](https://user-images.githubusercontent.com/1310400/117101150-387af980-ad7e-11eb-8572-e664f9f43baf.png)

**Reply by violenttamper** [reply]
原来是这样！我想知道tampermonkey扩展里在线安装的脚本，它可以在管理界面访问到安装的脚本网站，是不同的实现吗？

**Reply by violenttamper** [reply]
如果我手动更改了@homepage的网站，脚本下次更新会把我更改的网址改变吗？下次同步的时候会改变我改好的网址吗？

**Reply by gera2ld** [reply]
你可以在脚本的设置里面覆盖 `@homepage`，这个设置的优先级更高。
但是如果你修改了代码中的 `@homepage`，下次更新就会变回去。

**Reply by violenttamper** [reply]
请问，怎么“在脚本的设置里面覆盖 @homepage”？

**Reply by gera2ld** [reply]
Edit -> Settings -> Homepage URL
![image](https://user-images.githubusercontent.com/3139113/117106915-472fd380-adb3-11eb-8780-e03c7be636ff.png)

**Reply by violenttamper** [reply]
已解决，问题，感谢回复
